cypress/commons/utils/fileUtils.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+const parseMultipartFormData = (dataString) => {
+  const boundary = dataString.split('\r\n')[0];
+  let formParts = dataString.split(boundary);
+  formParts.pop();
+  formParts.shift();
+  formParts = formParts.map((el) => (el = el.replace(/\r\n$/, '')));
+  formParts = formParts.map((el) => (el = el.replace(/^\r\n/, '')));
```

Comment:
Not sure about it but could we simplify this map by `((el) => el.replace(/\r\n$/, ''))` ? :slightly_smiling_face:

Reply:
Oops you're right, this syntax was the leftovers of a first test with `forEach`, I'll change it :slightly_smiling_face: